clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/images/repo/FileResourcesRepo.kt

```diff
@@ -26,7 +26,7 @@ internal interface FileResourcesRepo {
     fun cleanupStaleFiles() = cleanupStaleFiles(emptyList())
     fun cleanupStaleFiles(urls: List<String>)
 
-    fun cleanupExpiredInAppsResources()
+    fun cleanupExpiredResources(cacheTpe: CtCacheType)
 
-    fun cleanupInAppsResources()
+    fun cleanupAllResources(cacheTpe: CtCacheType)
```

Comment:
note : the names were intentional since we are having such features. Passing a param introduces extra overhead of thinking of what the function does.

Reply:
Too many methods were creating a lot of confusion hence have to reduce to few with param. We will have comments on method which will convey what function does.